pom.xml

```diff
@@ -1004,12 +1004,16 @@
             <dependency>
                 <groupId>com.esri.geometry</groupId>
                 <artifactId>esri-geometry-api</artifactId>
-                <version>1.2.1</version>
+                <version>2.1.0</version>
                 <exclusions>
                     <exclusion>
                         <groupId>org.codehaus.jackson</groupId>
                         <artifactId>jackson-core-asl</artifactId>
                     </exclusion>
+                    <exclusion>
```

Comment:
The other exclusion on Jackson 1.x can probably be removed now.

Reply:
@electrum David, that's right. I'll do that. Thanks for pointing this out.